app/javascript/dashboard/components/widgets/conversation/MessagesView.vue

```diff
@@ -15,7 +14,9 @@ import Banner from 'dashboard/components/ui/Banner.vue';
 import { mapGetters } from 'vuex';
 
 // mixins
-import inboxMixin, { INBOX_FEATURES } from 'shared/mixins/inboxMixin';
+import { useInbox, INBOX_FEATURES } from 'shared/composables/useInbox';
+
+import aiMixin from 'dashboard/mixins/aiMixin';
```

Comment:
The `aiMixin` usage is already removed from the codebase. https://github.com/chatwoot/chatwoot/pull/9955
I guess there is some issue with the rebase.